### DISCUSSION TITLE
Key Ring of Github

### DISCUSSION THREAD

**Discussion by D3nan** [D3nan/Tafe24S1T102_Ext#10] [Q&A] [answered]
For Section C, where I'm getting everyone's public key, what is this 'key ring' and how do I add it?

When I go to SSH keys to add, the key I got from XG says it's already being used.

**Comment by SamuelTAFE** [comment, marked as answer]
It seems like that information might be redundant now?
(maybe repositories used to rely on a key ring to authenticate).

It's worked fine so far with us each having our own private/public PuTTY key pair, attached to our GitHub accounts.
